proposal/chrisPyr/README.rst

```diff
@@ -0,0 +1,118 @@
+================
+NSD Term Project
+================
+--------------------------
+chrisPyr @ 2021 NSD autumn
+--------------------------
+
+Basic Information
+=================
+
+Problem to Solve
+================
+
+| We all learned "Kepler's laws of planetary motion"[1] in high school.
+| The Kepler's laws can be explained and proved by Newtonian Mechanics[2].
+| Though the laws would fail to predict the orbits or speed of planets accurately in some certain conditions, e.g. Perihelion Motion of Mercury[3](fig. 1) and that's why we need "Theory of Relativity" to help us to describe and predict more precisely.
+| Einstein did prove this on his paper[4].
+| But that's not a point I need to worry about, Newtonian Mechanics should predict precise enough. I plan to build up a program that can predict the orbits of planets only take Newtonian Mechanics into account.
+| The kind of problem is actually a N-Body problem. And there is no general solution to N-Body problem, we can only try to simulate it with a numerical method.
+| The simplest idea is to compute all interation force between each body, which is a Brute-force method.
```

Comment:
I am not sure I understand what do you mean by the brute-force method.  To my understandings, there is not a close form solution to the problem.